example/app/components/LoginScreen.tsx

```diff
@@ -71,42 +39,41 @@ export const LoginScreen = () => {
           value={password}
           onChangeText={setPassword}
           secureTextEntry
-          autoCompleteType="password"
+          autoComplete="password"
           textContentType="password"
           placeholder="Password"
         />
       </View>
 
-      {authState === AuthState.LoginError && (
-        <Text style={[styles.error]}>
-          There was an error logging in, please try again
-        </Text>
-      )}
-      {authState === AuthState.RegisterError && (
+      {result.error && result.error.operation === AuthOperationName.LogIn && (
         <Text style={[styles.error]}>
-          There was an error registering, please try again
+          There was an error logging in, please try again{' '}
         </Text>
       )}
 
+      {result.error &&
+        result.error.operation === AuthOperationName.Register && (
```

Comment:
Is this a good place to use the `?.` operator?

```suggestion
      {result.error?.operation === AuthOperationName.Register && (
```